playwright/tests/screenshots/browsers.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect, type Page } from '@playwright/test';
+import { InscriptionView } from '../pageobjects/InscriptionView';
+import type { Section } from '../pageobjects/Section';
+import { screenshot } from './screenshot-util';
+import { Browser } from '../pageobjects/CodeEditor';
+
+const GENERIC_PID = {
+  SCRIPT: '168F0C6DF682858E-f3',
+  USER_TASK: '168F0C6DF682858E-f5',
+  DATABASE: '18141E75C9CEDD35-f3'
+} as const;
+
+test.describe('Browsers', () => {
+  test('ScriptingBrowser', async ({ page }) => {
+    await openSection(page, GENERIC_PID.SCRIPT, 'Output', 'Code').then(async section => {
+      const browser = await section.scriptArea().openBrowsers();
+      const dialog = browser.dialog();
+
+      await browser.openTab('Attribute');
+      await expect(browser.table().getByRole('row', { name: 'addPerson' }).first()).toBeVisible();
+      await screenshot(dialog, 'browser-attribute.png');
+
+      await browser.openTab('CMS');
+      await expect(browser.table().getByRole('row', { name: 'inscription-test-project' })).toBeVisible();
+      await screenshot(dialog, 'browser-cms.png');
+
+      await browser.openTab('Function');
+      await browser.search('role');
+      await expect(browser.table().getByRole('row', { name: 'ivy' })).toBeVisible();
+      await screenshot(dialog, 'browser-function.png');
+
+      await browser.openTab('Type');
+      await browser.search('Per');
+      await expect(browser.table().getByRole('cell', { name: 'Person :' })).toBeVisible();
+      await screenshot(dialog, 'browser-type.png');
+    });
+  });
+
+  test('RoleBrowser', async ({ page }) => {
+    await openSection(page, GENERIC_PID.USER_TASK, 'Task', 'Responsible').then(async section => {
+      await section.currentLocator().getByLabel('Browser').click();
+      const browser = new Browser(page);
+      await expect(browser.table().getByRole('row', { name: 'IT-Manager' })).toBeVisible();
+      await screenshot(browser.dialog(), 'browser-role.png');
+    });
+  });
+
+  test('DatabaseBrowser', async ({ page }) => {
+    await openSection(page, GENERIC_PID.DATABASE, 'Query', 'Condition').then(async section => {
+      await section.currentLocator().locator('.script-area').click();
+      await section.currentLocator().getByLabel('Browser').click();
```

Comment:
same as above + why not use the page object?
```ts
const code = section.scriptArea();
scriptArea.openBrowser();
```

Reply:
the section seems slightly different (it's not a classic 'code' section, but the query) ... so it doesn't work out of the box with the page object.